app/operations/organizations/destroy.rb

```diff
@@ -0,0 +1,12 @@
+module Organizations
+  class Destroy < Operation
+    integer :id
+
+    def execute
+      Organization.transaction do
+        organization = Organization.find(id)
+        Activation.disable_instances!([organization])
```

Comment:
As indicated, I reverted this change since it broke the shared spec. Updating this will be included when we have to rewrite all the controller specs during the Great Controller Refactoring that's perpetually just beyond the horizon.